src/lorawan_application_backend.erl

```diff
@@ -131,6 +132,12 @@ get_deveui(DevAddr) ->
         [] -> undefined
     end.
 
+get_appid(Profile) ->
+    case mnesia:dirty_read(profiles, Profile) of
```

Comment:
Did you test this? It should not work because the database name is `profile` (without `s`)